src/enums/RankingFilter.ts

```diff
@@ -0,0 +1,7 @@
+export const enum RankingFilter {
+  'Top 5' = 'Top5',
```

Comment:
Can you remove the spaces here? `RankingFilter['Top 5']` is more awkward to type than `RankingFilter.Top5`. Also can you please add the updated type to the README?

Reply:
Yea, sounds good! Will make another commit in a few mins.

Reply:
c8579e5 documents the changes to the `getPlayerRanking` options and adds a link to the new `RankingFilter` type